src/components/swimlanes/swimLanes.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  CardHeading,
+  Flex,
+  Heading,
+  SimpleGrid,
+  Spacer,
+  Text,
+  Link,
+} from "@nypl/design-system-react-components";
+import styles from "./Swimlanes.module.css";
+import React from "react";
+import { imageURL } from "@/utils/utils";
+import appConfig from "appConfig";
+
+const SwimLanes = ({ lanesWithNumItems }) => {
+  return (
+    <>
+      {lanesWithNumItems.map((lane, key) => (
+        <Box data-testid={lane.slug} key={lane.rank} mt="xxl">
+          <Flex alignItems="baseline">
+            <Heading id={`row-heading-${lane.slug}`} level="h2" size="heading3">
+              {lane.title}
+            </Heading>
+            <Spacer />
+            <Link
+              id={`row-see-more-${lane.slug}`}
+              type="standalone"
+              href={`${appConfig.DC_URL}collections/lane/${lane.slug}`}
+              aria-label={`See more ${lane.title.toLowerCase()}`}
+              sx={{
+                display: { sm: "none", md: "inline" },
+                fontWeight: "500",
+                alignItems: "center",
+              }}
+            >
+              See more
+            </Link>
+          </Flex>
+          {lane.collections && lane.collections.length > 0 && (
+            <SimpleGrid columns={4} id={`grid-${key}`}>
+              {lane.collections.map((collection, index) => (
+                <Card
+                  key={index}
+                  id={`card-${key}-${index}`}
+                  imageProps={{
+                    alt: "",
+                    aspectRatio: "twoByOne",
+                    src: imageURL(collection.image_id, "full", "288,", "0"),
+                  }}
+                >
+                  <CardHeading
+                    id={`row-card-heading-${key}-${index}`}
+                    level="h3"
+                    size="heading4"
+                    className={styles.collectiontitle}
+                    url={collection.url}
+                    noOfLines={3}
```

Comment:
@EdwinGuzman @bigfishdesign13 Stumbled on https://chakra-ui.com/docs/components/text/usage#truncate-text while I was working on the header, I think this is a way better solve for now even if doesn't address all the concerns in https://jira.nypl.org/browse/DSD-1625 ?

Reply:
Yes! that's helpful. Can you add this comment to the DSD-1625 ticket? The Chakra versions we are using is v1 so the link should be https://v1.chakra-ui.com/docs/components/typography/text

It's still available in v1 and we're moving to v2 soon.

Reply:
We are repurposing the Chakra `Text` component to compose our `Text` component, so the truncate props should work on our component as is.